src/pytest_env/plugin.py

```diff
@@ -2,14 +2,28 @@
 from __future__ import annotations
 
 import os
+import sys
 
 import pytest
 
+if sys.version_info >= (3, 8):  # pragma: no cover (py38+)
+    from typing import Final
+else:  # pragma: no cover (<py38)
+    from typing_extensions import Final
+
+_DEFAULT_FLAG: Final[str] = "D"
+_RAW_FLAG: Final[str] = "R"
+
+# Only used for testing purposes
+_TEST1_FLAG: Final[str] = "__PYTEST1__"
```

Comment:
I don't like these, can we remove please?

Reply:
okay